### PR TITLE
Add win64 build to version 6.10.2

### PR DESCRIPTION
**Pull request by swenger**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)t
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hadim** [comment]
@swenger I just created `6.10.x` out of `main`. Can you rebase your PR on it?

**Comment by hadim** [comment]
Also are you sure you need to build such an old version? See this comment on matrix: https://matrix.to/#/!SOyumkgPRWoXfQYIFH:matrix.org/$167457056443241pGznD:matrix.org?via=matrix.org&via=gitter.im&via=joachim.cc

**Comment by swenger** [comment]
> Also are you sure you need to build such an old version?

For some reason, `python-rocksdb` seems to depend on it: https://github.com/conda-forge/python-rocksdb-feedstock/pull/10/files#diff-bbefc910e37114b3d0d4542e18f264b70dfeab33b4fd57097ed3d9c4ad1971b3R18
If you have an idea why that is and how to fix it, I'd be happy to do that instead.

**Comment by swenger** [comment]
> @swenger I just created `6.10.x` out of `main`. Can you rebase your PR on it?

Done, thanks!

File: .scripts/build_steps.sh [hunk 1]
```diff
@@ -54,6 +54,7 @@ fi
 
 if [[ -f "${FEEDSTOCK_ROOT}/LICENSE.txt" ]]; then
   cp "${FEEDSTOCK_ROOT}/LICENSE.txt" "${RECIPE_ROOT}/recipe-scripts-license.txt"
+
```
Comment (swenger) [inline .scripts/build_steps.sh]:
```suggestion
```

**Comment by swenger** [comment]
> Also are you sure you need to build such an old version?

Looks like we can just update the pin instead to use the new version: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3971
Thanks @hadim and @xhochy!

**Pull request closed without merging**